README.md

```diff
@@ -2,6 +2,8 @@
 
 This repo contains R-shiny dashboards that are hosted on an R-shiny server.  Each subdir here contains a different dashboard, with dashboards accessible at https://shiny.covid.cloud.statcan.ca/<DASHBOARD_DIR_NAME>.
 
+**Note: Helping publish dashboards is a core piece of our service, but this particular R-Shiny hosting service may change over time.  We want to help anyone with *public* data that they want to share by a Shiny dashboard, but we cannot guarantee this exactly service will be here forever.**  Likely that means over time the URL and method of submitting/updating dashboards will change over time.
```

Comment:
Might change ever so slightly:

**Note: Helping publish dashboards is a core piece of our service, and it will be offered.**

**However, it might not be forever hosted at the `shiny.covid.cloud.statcan.ca` url, and the method of submitting/updating dashboards may change over time.**

Just want to make it VERY explicit that it'll always be offered

Reply:
Aside from that happy to merge